### PR TITLE
Todos

### PR DESCRIPTION
**Pull request by tmsolber**
Fixes @eirsyl suggested todos

### CODE REVIEW

File: brus/settings/production.py [hunk 1]
```diff
@@ -2,18 +2,17 @@
...
     DEBUG=(bool, False),
-    ALLOWED_HOSTS=(list, ['api.abakus.no']),
+    ALLOWED_HOSTS=(list, ['brus.abakus.no']),
 )
 environ.Env.read_env(os.path.join(os.path.dirname(BASE_DIR), '.env'))
 
 DEBUG = env('DEBUG')
 SECRET_KEY = env('SECRET_KEY')
-ALLOWED_HOSTS = env('ALLOWED_HOSTS')
+ALLOWED_HOSTS = []
```
Comment (eirsyl) [inline brus/settings/production.py]: You need to add ALLOWED_HOSTS = env('ALLOWED_HOSTS') here :)

File: brus/settings/development.py [hunk 2]
```diff
@@ -1,7 +1,7 @@
 from .base import INSTALLED_APPS, MIDDLEWARE_CLASSES
 
 DEBUG = True
-TEMPLATE_DEBUG = True
+TEMPLATE_DEBUG = False
```
Comment (eirsyl) [inline brus/settings/development.py]: This is deprecated, use the TEMPLATES dict instead

File: brus/settings/production.py [hunk 3]
```diff
@@ -25,13 +24,12 @@
     'default': env.cache()
 }
 
-# Auth
-ABAKUS_TOKEN = 'ffa15ee2b67710020838071b66927e6bb94351bb'
-
 AUTHENTICATION_BACKENDS = (
     'abakus.auth.AbakusBackend',
```
Comment (eirsyl) [inline brus/settings/production.py:28]: You should also add the ModelBackend here.

File: brus/urls.py [hunk 4]
```diff
@@ -1,26 +1,9 @@
...
-from django.conf.urls.static import static
-
 
 urlpatterns = [
     url(r'^admin/', admin.site.urls),
     url(r'^', include('brus.liste.urls')),
-    url(r'^login', 'django.contrib.auth.views.login', name='login'),
-] + static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)
+    url(r'^accounts/login', 'django.contrib.auth.views.login', name='login'),
+    url(r'^accounts/logout', 'django.contrib.auth.views.logout', name='logout'),
```
Comment (eirsyl) [inline brus/urls.py]: Maybe add {'next_page': '/'} here, the django logout page is ugly 😉

**Comment by eirsyl** [comment]
Looks, good! A frigg.yaml file and flake8 and isort inside tox would have been nice? Almost ready! ✨

File: brus/settings/development.py (hunk 2 above)
Comment (tmsolber) [inline brus/settings/development.py]: TEMPLATES dict is in base and default as False anyway. Can remove.

**Comment by eirsyl** [comment]
Is the migrations folder placed in /brus needed?

**Pull request merged**